Sources/SpeziViews/ViewModifier/ProcessingOverlay.swift

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+private struct ProcessingOverlay<Overlay: View>: ViewModifier {
+    fileprivate var isProcessing: Bool
+    @ViewBuilder fileprivate var overlay: () -> Overlay
+
```

Comment:
Please add two empty lines between similar type elements like stored properties and, e.g., functions. I should somehow write this down with some other implicit code style guidelines that I normally use ...
```suggestion


```